vm/install-cmo-gxargparse.sh

```diff
@@ -13,7 +13,7 @@ DEST_PATH="/usr/local/bin/"
 # install cmo
 
 # get cmo from git
-sudo git clone https://github.com/mskcc/cmo.git ${DEST_PATH}/cmo-gxargparse/cmo
+sudo git clone https://github.com/ionox0/cmo.git ${DEST_PATH}/cmo-gxargparse/cmo
 sudo chown -R vagrant:vagrant ${DEST_PATH}/cmo-gxargparse/cmo
 
 # install dependencies
```

Comment:
I changed this line so that I could test the changes that I made to the `cmo` repo, while running the tool from roslin. 

Again, this I feel might need to be kept separate. I'm not 100% of the best way to maintain these changes.

Reply:
This also means that I was committing straight to the cmo `master` branch, which is obviously not ideal.

I think I might be able to do something like the follwing:
```
#---------------------------------
# install cmo

# get cmo from git
sudo git clone https://github.com/ionox0/cmo.git ${DEST_PATH}/cmo-gxargparse/cmo
sudo chown -R vagrant:vagrant ${DEST_PATH}/cmo-gxargparse/cmo

# install dependencies
sudo pip install --upgrade pip

# install
cd ${DEST_PATH}/cmo-gxargparse/cmo
python setup.py develop --user
```

but this might also be something that would benefit from some external configuration or env vars instead of hardcoding a separate fork/branch of `cmo`.